app/controllers/api/trainees_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Api
+  class TraineesController < Api::BaseController
+    def show
+      trainee = Trainee.find_by(slug: params[:id])
```

Comment:
We need to use something like `current_provider.trainees.find_by...` here to make sure all data is scoped to the authenticated provider. (Even though the authentication mechanism isn't ready yet!)

Reply:
Yeah I wasn't sure how far to go into that, since there's no authentication as you say. But I'll put something like that in so we don't forget.

Reply:
@stevehook I've added a placeholder `current_provider` method to the base class, which basically does nothing for now. The authentication ticket can flesh out all of those methods separately. Let me know what you think.